src/components/Hero/Hero.module.css

```diff
@@ -0,0 +1,72 @@
+.hero {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  padding: 4rem 0;
+  width: 100%;
+}
+
+.hero__wrapper-title {
+  font-size: 3.5rem;
+  padding: 0;
+  margin: 0;
+}
+
+.hero__wrapper-description {
+  max-width: 30rem;
+  font-size: 1.125rem;
+  line-height: 1.875rem;
+  padding: 0;
+  margin: 1rem 0;
+}
+
+.hero__image {
+  width: 30rem;
+  animation: spin infinite 100s linear;
```

Comment:
At smaller screen (~500px) the animation causes overflow on the y-axis. This only happens on certain frames since the image is rotated in a way that the corner wants to appear out of the screen.

Reply:
Resized the image to a smaller value, so that the image won't overflow now on smaller screens.